Remotion/ObjectBinding/Web/UI/Controls/BocEnumValue.cs

```diff
@@ -445,7 +445,15 @@ public override Control TargetControl
     [Browsable(false)]
     public string? FocusID
     {
-      get { return IsReadOnly ? null : GetValueName(); }
+      get
+      {
+        if (IsReadOnly)
+          return null;
+        if (ListControlStyle.ControlType == ListControlType.RadioButtonList)
+          return Controls.Count > 0 ? GetValueName() + "_0" : null;
```

Comment:
Please add a comment that the ID is calculated by System.Web.U.ListBox. Some day I want to rewrite the rendering without relying on ASP.NET WebForms, then we could conceivably remove this duplication.